fhir-server/liberty-config/config/default/fhir-server-config-postgresql.json

```diff
@@ -49,22 +49,16 @@
             "jdbc": {
                 "__comment": "Configuration properties for the JDBC persistence implementation",
                 "bootstrapDb": true,
-                "dataSourceJndiName": "jdbc/fhirProxyDataSource",
+                "enableProxyDatasource": false,
+                "bootstrapDataSourceBase": "jdbc/bootstrap",
                 "enableCodeSystemsCache": true,
                 "enableParameterNamesCache": true,
                 "enableResourceTypesCache": true
             },
             "datasources": {
                 "default": {
                     "type": "postgresql",
-                    "connectionProperties": {
-                        "serverName": "postgres_postgres_1",
-                        "portNumber": 5432,
-                        "databaseName": "fhirdb",
-                        "user": "fhirserver",
-                        "password": "change-password",
-                        "currentSchema": "fhirdata"
-                    }
+                    "currentSchema": "fhirdata"
```

Comment:
Doesn't feel so much like deprecating, rather this is a breaking change.

Reply:
This is following the new pattern, which requires a configuration change.

Reply:
Old configuration will continue to work.